docs/resources/dds_instance.md

```diff
@@ -235,17 +237,24 @@ The `flavor` block supports:
 
 The `backup_strategy` block supports:
 
-* `start_time` - (Required, String) Specifies the backup time window. Automated backups will be triggered during the
-  backup time window. The value cannot be empty. It must be a valid value in the
-  "hh:mm-HH:MM" format. The current time is in the UTC format.
+* `start_time` - (Required, String) Specifies the backup time window. Automated backups will be triggered during
+  the backup time window. The value cannot be empty. It must be a valid value in the "hh:mm-HH:MM" format.
+  The current time is in the UTC format.
   + The HH value must be 1 greater than the hh value.
-  + The values from mm and MM must be the same and must be set to any of the following 00, 15, 30, or 45.
+  + The values from mm and MM must be the same and must be set to **00**.
 
 * `keep_days` - (Required, Int) Specifies the number of days to retain the generated backup files. The value range is
-  from 0 to 732.
-  + If this parameter is set to 0, the automated backup policy is not set.
-  + If this parameter is not transferred, the automated backup policy is enabled by default. Backup files are stored
-      for seven days by default.
+  from 0 to 732. If this parameter is set to 0, the automated backup policy is disabled.
+
+* `period` - (Optional, String) Specifies the backup cycle configuration. Data will be automatically backed up on the
+  selected days every week.
+  Value range: The value is a list of digits separated by commas (,). Each digit indicates a day of the week. The
+  restrictions on the backup retention period are as follows:
+  + This parameter is no need if the retention period is set to 0.
+  + If you set the retention period to 1 to 6 days, data is automatically backed up each day of the week. Set the
+    parameter value to 1,2,3,4,5,6,7.
+  + If you set the retention period to 7 to 732 days, select at least one day of the week for the backup cycle. Example
+    value: 1,2,3,4.
```

Comment:
this description is confusing, please make it more clear

Reply:
done